### DISCUSSION TITLE
Les pingouins

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#54] [Q&A]
Commentaires sur ce quiz. 

**Accès**
- Serveur de développement (protégé) : https://timunix3.csfoy.ca/~quiz-TIM/rodrigorafaelgaray/index.html
- Dépôt Github (public): https://github.com/rodrigorafaelgaray/quiz-tim
- Github.io : https://rodrigorafaelgaray.github.io/quiz-tim/rodrigorafaelgaray/index.html

**Comment by Heyliya** [comment]
On est pas capble de répondre au quiz sans la sourie. Il n'est pas accessible.
Il aurais pus avoir plus de transitions des boutons.
les boutons de choix de réponses devrais avoir un design desactivé lors de la rétroaction.
La section résultat devrais être ajouter en JS

Le design est très simple et fite parfaitement avec le sujet
Par Léa Marois

**Comment by raphaellabbelatreille** [comment]
Sans surprise, il y a beaucoup d'image de pingouins dans ce quiz. La question 2 par contre, n'aurais pas besoin d'images pour les choix de réponse comme ce sont des durée en minutes les réponses.
En général, le site est très bien fait (choix de couleurs et animation sur les bouton qui change de couleurs) mais les zones de rétroaction suite à la réponse me semble mal placé. Selon moi, il aurait pu être mieux stylisé pour mieux s'accordé au reste.
La barre de progression qui apparait en haut à droite est très bien pensé et j'aime comment, sur la section résultat, on nous met un rappel sur quelle question on a eu bon ou mauvais.
Le logo est mignon.

Par Raphaël Labbé-Latreille

**Comment by kateundercoffler** [comment]
Moi j'ai bien aimé ton quiz.  Le sujet est tellement intéressant et j'avais appris quelque chose :)  Bien fait.  Ta pallette de couleurs sont belles, je suggère de peut-être faire la texte un peu plus noir pour avoir un peu plus de contrast.  Oui, j'écho les commentaires concernant la sourie.  En général, bravo et bon travail !

**Comment by evefevrier** [comment]
@rodrigorafaelgaray 
Bravo ce quiz est instructif et bien réalisé, mention spéciale pour la page de résultats et l'ajout du bouton recommencer. 🥇 En + tu l'as publié sur Github.io 🚀
